src/cow-react/modules/limitOrders/pure/Orders/OrderRow/index.tsx

```diff
@@ -198,12 +202,23 @@ export function OrderRow({
   // const executedTimeAgo = useTimeAgo(expirationTime, TIME_AGO_UPDATE_INTERVAL)
   const activityUrl = chainId && activityId ? getEtherscanLink(chainId, activityId, 'transaction') : undefined
 
-  const executionPriceInversed = isRateInversed
-    ? prices?.estimatedExecutionPrice.invert()
-    : prices?.estimatedExecutionPrice
-  // const marketPriceInversed = isRateInversed ? prices?.marketPrice.invert() : prices?.marketPrice
-  const executedPriceInversed = isRateInversed ? executedPrice?.invert() : executedPrice
-  const spotPriceInverted = isRateInversed ? spotPrice?.invert() : spotPrice
+  const [isInverted, setIsinverted] = useState(isRateInverted)
+
+  // Update internal isInverted flag whenever prop change
+  useEffect(() => {
+    setIsinverted(isRateInverted)
```

Comment:
Shouldn't this be !isInverted 

haven't tested, but my reasoning:
- I see you added local state for isInverted in the row
- I see that the same thing is coming from a prop

I deduct:
- The row can invert (i guess this is when u click)
- But also, is possible that there's a "general is inverted" for when you invert from the header

My intuition...
- If you updated the row
- this info will be lost if you invert the column. 
- If inverting the column means, i want to invert all the rows... then we need to use !isInverted instead of isRateInverted

Reply:
That's true, the smart quote will be applied only on load.
On first click, the parent logic will be applied, overwriting local state.
This behaviour was already in place for limit orders and I just extended it for the whole table.

I had not considered/tested the row inverting, only the whole table.
Will have to work on that